src/ssl.c

```diff
@@ -985,8 +985,10 @@ int wolfSSL_SetTmpDH(WOLFSSL* ssl, const unsigned char* p, int pSz,
     if (pSz < ssl->options.minDhKeySz)
         return DH_KEY_SIZE_E;
 
+    #ifndef WOLFSSL_WPAS
```

Comment:
I am making hostapd work as well as it uses the same TLS code and you can test wpa_supplicant against it. A WOLFSSL object can be turned into either a client or server. The generic TLS using wolfSSL code sets the DH parameters regardless of whether it is a client or server.